research/GPT/requirements.txt

```
@@ -0,0 +1,4 @@
+torch
+sparseml
+transformers
+numpy
```

nit: `sparseml[torch]` will make sure a compatible version of pytorch is installed, and numpy should come with them
```suggestion
sparseml[torch]
transformers
```